front/app/resources/GetCampaignDeliveries.tsx

```diff
@@ -72,14 +72,19 @@ export default class GetCampaignDeliveries extends React.Component<
           })
         )
         .subscribe((deliveries) => {
+          const currentPageNumberFromURL = !isNilOrError(deliveries)
```

Comment:
This is needed because of the parameter type that's introduced for getPageNumberFromUrl.

Reply:
(And it does exactly the same as before. If the arg passed to getPageNumberFromUrl was not a string, it returned null. Now we're just doing the type check before passing the function parameter, not in the function body.)